src/tutorial/scheduling/integer-cycling.rst

```diff
@@ -362,46 +362,33 @@ Recurrence Sections
            # Repeat every third cycle.
            P3 = baz
 
-.. digraph:: example
+.. image:: ../../img/reccurrance-sections.svg
    :align: center
 
-   subgraph cluster_1 {
-       label = 1
-       style = dashed
-       "foo.1" [label="foo\n1"]
-       "bar.1" [label="bar\n1"]
-       "baz.1" [label="baz\n1"]
-   }
-
-   subgraph cluster_2 {
-       label = 2
-       style = dashed
-       "foo.2" [label="foo\n2"]
-   }
-
-   subgraph cluster_3 {
-       label = 3
-       style = dashed
-       "foo.3" [label="foo\n3"]
-       "bar.3" [label="bar\n3"]
-   }
-
 .. nextslide::
 
 .. ifnotslides::
 
-   By default recurrences start at the :term:`initial cycle point`, however it
-   is possible to make them start at an arbitrary cycle point. This is done by
-   writing the cycle point and the recurrence separated by a forward slash
-   (``/``), e.g. ``5/P3`` means repeat every third cycle starting *from* cycle
-   number 5.
+   You may have noticed that in the previous example that it is not possible to
```

Comment:
Can you delete this first sentence?  It could be misinterpreted, on a quick read, as a definitive statement.  I'd just start the next sentence with "To get only even cycle points, start the recurrence from an even point..." (or similar).